scripts/ci-breadcrumbs.js

```
@@ -0,0 +1,247 @@
+#!/usr/bin/env node
+/* eslint-env node */
+/* global module, require, process, console */
```

ESLint

[error] 'module' is already defined as a built-in global variable.

(no-redeclare)